vllm/v1/engine/async_llm.py

```diff
@@ -23,6 +23,7 @@
 from vllm.v1.engine.detokenizer import Detokenizer
 from vllm.v1.engine.processor import Processor
 from vllm.v1.executor.abstract import Executor
+from vllm.v1.executor.ray_utils import initialize_ray_cluster
```

Comment:
This should be a lazy import right? Doesn't this import ray?

Reply:
This will import Ray but won't error out if Ray is not installed:

```python
try:
    import ray
    ...
except ImportError:
    ray = None
```

It errors out only when `initialize_ray_cluster ` is called:

```python
def initialize_ray_cluster():
    assert_ray_available()
```